Rebus.AdoNet/AdoNetExtensions.cs

```diff
@@ -62,5 +62,19 @@ public static AdoNetTimeoutStorageFluentConfigurer StoreInAdoNet(this RebusTimeo
 
 			return storage;
 		}
+
+		/// <summary>
+		/// Configures Rebus to store timeouts in AdoNet with limit for GetDueTimeouts.
+		/// </summary>
+		public static AdoNetTimeoutStorageFluentConfigurer StoreInAdoNet(this RebusTimeoutsConfigurer configurer, string connectionStringName, string timeoutsTableName, uint limit)
```

Comment:
Rename 'limit' to batchSize or something similiar, and please, add documentation to the method explaining what limit (or batchSize) is for..